integration/test/basic/main_test.go

```diff
@@ -8,55 +8,48 @@ import (
 	"os"
 	"testing"
```

Comment:
Did we have a test that wasn't ran here?

Reply:
The test steps are the same. The only difference is `managedservices` uses appr and `basicapp` uses helm repositories.

Reply:
Was it copy/paste from another repo?

Reply:
I think this is because Ji removed the circle config when he copied the code from the kubernetes-node-exporter repo.

See https://github.com/giantswarm/node-exporter-app/pull/1/files#r330138218.

Is that what you mean about the test not being run?

Reply:
Yes. Thanks.